woocommerce/woocommerce-android
# Activate Gift card feature flag

Summary
==========
Activates the Gift card feature flag effectively landing the feature to all our users.

How to Test
==========
Simply make sure that everything is working properly with the Gift card extension following all latest PRs.

Update release notes:
